js/utils/index.js

```diff
@@ -142,7 +143,12 @@ export function isValidBitcoinAddress(address) {
     bitcoreLib.encoding.Base58Check.decode(address);
     return true;
   } catch (exc) {
-    return false;
```

Comment:
What you have there will always throw and exception since if something is base58 encoded it can't also be bech32 encoded and vice versa.